deploy/KedaScaleController.yaml

```diff
@@ -82,7 +82,7 @@ spec:
       serviceAccountName: keda-operator
       containers:
       - name: keda-operator
-        image: kedacore/keda:latest
+        image: mahmuds2/scaler:latest
```

Comment:
@t-shama, @Aarthisk, @cgillum I think the images for kedacore should come from kedacore repo so that it is consistent across the board. What do you guys think?

Reply:
I changed it back to kedacore! I was originally using my own docker repo to test my scaler and forgot to change it back.